sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ChangeFeedProcessorContext.java

```diff
@@ -25,4 +25,15 @@ public interface ChangeFeedProcessorContext {
      * */
     @Beta(value = Beta.SinceVersion.V4_51_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
     String getLeaseToken();
+
+    /**
+     * Get the diagnostics from the underlying feed response.
+     *
+     * @return The diagnostics object.
+     */
+    @Beta(value = Beta.SinceVersion.V4_64_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+    default CosmosDiagnostics getDiagnostics()
```

Comment:
nit: Java indentation convention is to open braces in the same line.

Reply:
fixed